### DISCUSSION TITLE
Support for PS4 controller

### DISCUSSION THREAD

**Discussion by JudeAbbey** [pybricks/support#1736] [General]
Is there support for any other remote controller beside xbox? I have PS4 and logitec F310 that I do like to control a spike prime.

Thanks

**Comment by laurensvalk** [comment]
Just the Xbox Controller, because it supports Bluetooth Low Energy.

Controllers like the PS4 use Bluetooth classic, which doesn't work.

**Reply by JudeAbbey** [reply]
Got it. Thanks
